app/styles/app.css

```diff
@@ -47,6 +47,25 @@ body {
   margin-top: 50px;
 }
 
+.qr-code-container{
+  display: flex;
+  justify-content: center;
+  margin-top: 15px;
+}
+
+.qr-code{
+  width: 200px ;
+  height: 200px;
+  margin-top: 50px;
+  display: flex;
+  text-align: center;
+}
+
+.qr-code-text{
+  text-align: center;
+  margin-top:15px;
+}
+
```

Comment:
Can we please create a new CSS file for the new template?

Reply:
Can we please remove the duplicated CSS from here ? and the filename could be same as route name as per our general convention for component and template specific CSS.

For components - CSS file name should be same as component name
For templates - CSS file name should be same as template name